### ISSUE TITLE
Importing fails with latest npm package 1.2.0

### ISSUE THREAD

**Issue by mhp-borisbojic** [hujiulong/vue-3d-model#127]
@hujiulong 

Sadly, the new npm package v1.2.0 doesn't work anymore at all:

```
This dependency was not found:

* vue-3d-model in ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/App.vue?vue&type=script&lang=js&

To install it, you can run: npm install --save vue-3d-model
```

I even tried to use it in a completely fresh vue project - the error message remains.

Reverting to v1.1.1 solved it again.

**Comment by hujiulong** [comment]
1.2.0 is the first version that uses GitHub actions to publish automatically. It seems that actions script has some bugs. I manually released 1.2.1 and tried to install it.

**Comment by mhp-borisbojic** [comment]
Good morning,

the npm package works again so far, but providing the new **controlsOptions** creates an error:

```
[Vue warn]: Error in mounted hook: "TypeError: Object.assing is not a function"
```

And:

```
TypeError: "Object.assing is not a function"
    updateControls vue-3d-model.umd.js:54101
    update vue-3d-model.umd.js:53989
    mounted vue-3d-model.umd.js:53870
    VueJS 12
```

My call:

```
      <model-gltf
        src="/shoe3d_assets/objects/Wing2_optimized.gltf"
        :backgroundAlpha="0"
        :rotation="rotation"
        :controlsOptions="controlsOptions"
      />
```

And options:

```
  data: function() {
    return {
      rotation: {
        x: 0,
        y: -1.8,
        z: 0
      },
      controlsOptions: {
        enablePan: false,
        enableZoom: false
      }
    }
  },
```

**Comment by hujiulong** [comment]
I released 1.2.2 to fix this bug.
[example](https://hujiulong.github.io/vue-3d-model/#/demo-controls)

**Comment by mhp-borisbojic** [comment]
Thank you! Works perfectly now :)
